nwapp/utils/notifier.js

```diff
@@ -28,14 +36,85 @@ function playNotificationSound() {
   audio = null;
 }
 
+
+// Cache lookup
+var urlFilePathMap = new Map();
+function getFileAtUrl(url) {
+  return new Promise(function(resolve, reject) {
+    var cachedFilePath = urlFilePathMap.get(url);
+    if(cachedFilePath) {
+      resolve(cachedFilePath);
+    }
+    else {
+      setTimeout(function() {
+        reject(new Error('Took to long to request file', url));
+      }, 750);
+
+      var tempFileStream = temp.createWriteStream();
```

Comment:
@trevorah The `temp.track()` call keeps track of the files and cleans up on exit: https://github.com/gitterHQ/desktop/pull/100/files#diff-d74e5f0a212623b40b7c84b3d6313d78R6

I tested and works fine on Windows.